roles/lib_openshift/src/class/oc_obj.py

```diff
@@ -50,7 +50,11 @@ def create(self, files=None, content=None):
         if files:
             return self._create(files[0])
 
-        content['data'] = yaml.dump(content['data'])
+        if hasattr(yaml, 'RoundTripDumper'):
+            content['data'] = yaml.dump(content['data'], Dumper=yaml.RoundTripDumper)
```

Comment:
Without context I find this easily becoming confusing in the future. Would you mind adding a comment explaining why this conditional must exist?